test/test_pebble.py

```diff
@@ -2702,6 +2750,124 @@ def test_checklevel_conversion(self):
             ('GET', '/v1/checks', {'level': 'ready', 'names': ['chk2']}, None),
         ])
 
+    def test_get_notice(self):
+        self.client.responses.append({
+            'result': {
+                'id': '123',
+                'user-id': 1000,
+                'type': 'custom',
+                'key': 'example.com/a',
+                'first-occurred': '2023-12-07T17:01:02.123456789Z',
+                'last-occurred': '2023-12-07T17:01:03.123456789Z',
+                'last-repeated': '2023-12-07T17:01:04.123456789Z',
+                'occurrences': 7,
+            },
+            'status': 'OK',
+            'status-code': 200,
+            'type': 'sync',
+        })
+
+        notice = self.client.get_notice('124')
+
+        # No need to re-test full Notice.from_dict behaviour
```

Comment:
My preference for comments is that they are a full sentence, including final punctuation, so:

```suggestion
        # No need to re-test full Notice.from_dict behaviour.
```

However, it's hard to get a smaller nit than this, and I don't think this is consistent in the rest of ops, so feel free to discard this comment :smile:.

Reply:
Fair, updated.